### DISCUSSION TITLE
Filament Splicer consideration

### DISCUSSION THREAD

**Discussion by jburleson795** [Enraged-Rabbit-Community/ERCF_v2#120] [Ideas]
I am new to this whole thing, but was wondering if there was any thoughts about adding a splicer just beyond the cutter. Something like this could work and would reduce wasted filament if dialed in enough to purge in the infill

https://www.thingiverse.com/thing:6116350

Just curious if anyone was working on this. I would like to, but I am very new to all of this and will need some help with the software side once its put together.

**Comment by norris108** [comment]
Adding a device such as that would introduce drag into the system, and with ERCF you want the least amount of drag possible to prevent putting extra stress on the ERCF and extruder stepper motors and to allow filament to be fed without any resistance
